### PR TITLE
Add Cloud Security Center v1 API.

### PR DESCRIPTION
**Pull request by kolea2**
Please double check pom.xml's and versions.txt. Thanks!

### CODE REVIEW

**Comment by sduskis** [comment]
@kolea2, please run the formatter.

File: google-cloud-bom/pom.xml [hunk 1]
```diff
@@ -620,6 +620,16 @@
         <artifactId>grpc-google-cloud-securitycenter-v1beta1</artifactId>
         <version>0.47.1-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-securitycenter-v1beta1:current} -->
       </dependency>
+      <dependency>
```
Comment (sduskis) [inline google-cloud-bom/pom.xml]: Please also add a reference to `google-cloud-clients/google-cloud-securitycenter`

File: versions.txt [hunk 2]
```diff
@@ -91,6 +91,7 @@ grpc-google-cloud-redis-v1:0.47.0:0.47.1-SNAPSHOT
 grpc-google-cloud-redis-v1beta1:0.47.0:0.47.1-SNAPSHOT
 grpc-google-cloud-scheduler-v1beta1:0.47.0:0.47.1-SNAPSHOT
 grpc-google-cloud-securitycenter-v1beta1:0.47.0:0.47.1-SNAPSHOT
+grpc-google-cloud-securitycenter-v1:0.47.0:0.47.1-SNAPSHOT
```
Comment (sduskis) [inline versions.txt]: Please also add a version for google-cloud-clients:google-cloud-securitycenter

Comment (kolea2) [reply versions.txt]: It's already there - https://github.com/googleapis/google-cloud-java/pull/4659/files/1175022db1edab863af7789df5b3fb11fe22b67d#diff-9fa081a8634fe9e631cd85762bb9ceb9R52

File: google-cloud-bom/pom.xml (hunk 1 above)
Comment (kolea2) [reply google-cloud-bom/pom.xml]: Fixed

**Pull request merged**